### PR TITLE
Add support for CORS

### PR DESCRIPTION
**Pull request by escalona**
Addresses #31

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: lib/fetchr-plugin.js [hunk 1]
```diff
@@ -92,8 +96,9 @@ module.exports = function fetchrPlugin(options) {
                         'delete': crudProxy(service, 'delete', serviceMeta),
                         constructGetXhrUri: function constructGetXhrUri(resource, params, config) {
                             config = config || {};
+                            uri = config.cors ? corsPath : xhrPath;
```
Comment (Vijar) [inline lib/fetchr-plugin.js]: This line is duplicating logic[1] from fetchr now, but I don't think there is any better way to abstract it as we already implemented the `constructGetXhrUri` function. @lingyan thoughts?

[1] https://github.com/yahoo/fetchr/blob/65bab4053b73ae8451dff232db1bc294dcb2dda5/libs/fetcher.client.js#L342

Comment (escalona) [inline lib/fetchr-plugin.js]: Noticed that as well. Only thing I could think of is to create a new utility module that takes config, corsPath, and xhrPath but thought that was a bit overkill.

Comment (Vijar) [inline lib/fetchr-plugin.js]: I think for now, we should get this in to fix the bug. I'll create a separate issue to track this.

**Comment by Vijar** [comment]
:ship:

**Pull request merged**